### DISCUSSION TITLE
I want to add skeleton every time recycling is waiting to re-render item.

### DISCUSSION THREAD

**Discussion by LuckyBoy124** [Shopify/flash-list#836] [Q&A]
## Description
Currently every time I swipe to the end of the flash list and then I quickly swipe up it shows a space like the video below. So what should I do if I want to add skeletons? 
## Video
https://user-images.githubusercontent.com/108321684/236413223-bbf26b5b-9108-4bfa-b221-9b79e7d436bd.mp4

**Comment by Bayramito** [comment]
i am also wondering about that...

btw how did you achieve this fade in animation on your items?
